src/docfx/build/moniker/MonikerProvider.cs

```diff
@@ -112,7 +130,7 @@ public int GetMonikerOrder(string moniker)
                 if (!_config.SkipMonikerValidation && configMonikerRange.Value is null)
                 {
                     errors.Add(Errors.Versioning.MonikerRangeUndefined(metadata.MonikerRange.Source));
-                    return (errors, configMonikers);
+                    return (errors, configMonikers.Except(excludeMonikers));
```

Comment:
should `exclude_monikers` be processed _before_ checking intersection? Aka, should we report error when a file produces an empty moniker list due to `exclude_monikers`?